### PR TITLE
JAMES-3028 define implementation in blob properties for cassandra docker images

### PR DESCRIPTION
**Pull request by remk**

### CODE REVIEW

File: dockerfiles/run/guice/cassandra-ldap/destination/conf/blob.properties [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
+# ============================================= BlobStore Implementation ==================================
+# Read https://james.apache.org/server/config-blobstore.html for further details
 
-# Choosing blob exporting mechanism, allowed mechanism are: localFile, linshare
-# LinShare is a file sharing service, will be explained in the below section
-# Optional, default is localFile
-blob.export.implementation=localFile
+# Choose your BlobStore implementation
+# Mandatory, allowed values are: cassandra, s3
+implementation=cassandra
```
Comment (chibenwa) [inline dockerfiles/run/guice/cassandra-ldap/destination/conf/blob.properties:6]: Isn't the implementation always Cassandra in cassandra-guice product ?

**Review by chibenwa — COMMENTED** [review]
JAMES-3028 as a ticket number?

Also, we should use rabbitmq in dockercomposes (just in case)

File: dockerfiles/run/guice/cassandra-ldap/destination/conf/blob.properties (hunk 1 above)
Comment (remk) [reply dockerfiles/run/guice/cassandra-ldap/destination/conf/blob.properties:6]: I can only speak for the openpaas image. But if no blob.properties is provided james start successfully, if a blob.properties is provided without 'implementation' it fails with a message pointing out this problem.

**Review by chibenwa — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Comment by Arsnael** [comment]
```
[eae500f30f6e5fb646dd03b0a2ca033ed04981bd] [ERROR] Failures: 
[eae500f30f6e5fb646dd03b0a2ca033ed04981bd] [ERROR]   CassandraMailboxMapperTest$ConsistencyTest.createAfterPreviousDeleteOnFailedCreateShouldCreateAMailbox:604 Multiple Failures (1 failure)
[eae500f30f6e5fb646dd03b0a2ca033ed04981bd] 	
[eae500f30f6e5fb646dd03b0a2ca033ed04981bd] Expected size:<1> but was:<2> in:
[eae500f30f6e5fb646dd03b0a2ca033ed04981bd] <[Mailbox{id=09b101e0-0eef-11eb-b2d8-1dc38d1e95a2, namespace=#private, user=Username{localPart=user, domainPart=Optional.empty}, name=name},
[eae500f30f6e5fb646dd03b0a2ca033ed04981bd]     Mailbox{id=22451f20-0eef-11eb-b2d8-1dc38d1e95a2, namespace=#private, user=Username{localPart=user, domainPart=Optional.empty}, name=INBOX}]>
[eae500f30f6e5fb646dd03b0a2ca033ed04981bd] at CassandraMailboxMapperTest$ConsistencyTest.lambda$createAfterPreviousDeleteOnFailedCreateShouldCreateAMailbox$34(CassandraMailboxMapperTest$ConsistencyTest.java:615)
```
test this please

**Comment by chibenwa** [comment]
I fixed that test in another PR ^^

**Comment by Arsnael** [comment]
> I fixed that test in another PR ^^

Do you mean master is broken or it is a flaky test you fixed somewhere else? :)

**Comment by chibenwa** [comment]
it is a flaky test I fixed somewhere else

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**